scripts/lib-franklin.js

```diff
@@ -225,6 +225,46 @@ export async function decorateIcons(element) {
   });
 }
 
+export async function decorateTags(element) {
```

Comment:
Why lib-franklin and not scripts.js? I would leave lib-franklin.js untouched (although I see we have changed it a bit...)

Reply:
I chose lib-franklin instead of scripts.js because all of the other decorate functions are in this file such as for example: decorateButtons